components/Caretaker.js

```diff
@@ -53,7 +52,7 @@ class Caretaker extends React.Component {
           leftComponent={{ icon: 'home', color: '#fff', size: normalize(40), onPress:() => this.props.navigation.navigate('Home')} }
           rightComponent={{ icon: 'add', color: '#fff', size: normalize(40), onPress:() => this.setIsVisible(true)} }
         />
-        <View style={{marginBottom: 160}}>
+        <View style={styles.viewArea}>
```

Comment:
Nice! I meant to mention this earlier when we were talking about it. Hopefully you found it pretty easy.

Reply:
I think this is a good feature. nice quick look at the status of tasks for the provider